### PR TITLE
add mv1 enhance backbone

### PR DESCRIPTION
**Pull request by littletomatodonkey**
- [x] add mv1 enhance backbone
- [x] add distillation doc

### CODE REVIEW

File: doc/doc_ch/knowledge_distillation.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        name: SequenceEncoder
+        encoder_type: rnn
+        hidden_size: 64
+      Head:
+        name: CTCHead
+        mid_channels: 96
+        fc_decay: 0.00002
+```
+
+当然，这里如果希望添加更多的子网络进行训练，也可以按照`Student`与`Teacher`的添加方式，在配置文件中添加相应的字段。蒸馏模型`DistillationModel`类的具体实现代码可以参考[distillation_model.py](../../ppocr/modeling/architectures/distillation_model.py)。
```
Comment (WenmuZhou) [inline doc/doc_ch/knowledge_distillation.md:92]: 建议这里给个例子

File: ppocr/modeling/backbones/rec_mv1_enhance.py [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+
+    def forward(self, inputs):
+        y = self._depthwise_conv(inputs)
+        if self.use_se:
+            y = self._se(y)
+        y = self._pointwise_conv(y)
+        return y
+
+
+class MobileNet(nn.Layer):
```
Comment (WenmuZhou) [inline ppocr/modeling/backbones/rec_mv1_enhance.py:112]: MobileNetV1

File: configs/rec/ch_ppocr_v2.1/rec_chinese_lite_train_distillation_v2.1.yml [hunk 3]
```diff
@@ -32,51 +32,49 @@ Optimizer:
...
       pretrained:
       freeze_params: false
       return_all_feats: true
-      model_type: rec
+      model_type: *model_type
       algorithm: CRNN
       Transform:
       Backbone:
-        name: MobileNetV3
+        name: MobileNet
```
[the remark this answers is not in the document]
Comment (littletomatodonkey) [reply configs/rec/ch_ppocr_v2.1/rec_chinese_lite_train_distillation_v2.1.yml]: done, txh

File: configs/rec/ch_ppocr_v2.1/rec_chinese_lite_train_distillation_v2.1.yml [hunk 4]
```diff
@@ -32,51 +32,49 @@ Optimizer:
...
```
[the remark this answers is not in the document]
Comment (littletomatodonkey) [reply configs/rec/ch_ppocr_v2.1/rec_chinese_lite_train_distillation_v2.1.yml]: done

File: ppocr/modeling/backbones/rec_mv1_enhance.py (hunk 2 above)
Comment (littletomatodonkey) [reply ppocr/modeling/backbones/rec_mv1_enhance.py]: done

File: doc/doc_ch/knowledge_distillation.md (hunk 1 above)
Comment (littletomatodonkey) [reply doc/doc_ch/knowledge_distillation.md]: done, thx.

**Review by WenmuZhou — APPROVED** [review]

**Pull request merged**